lib/client-assets.php

```diff
@@ -181,6 +193,7 @@ function gutenberg_register_packages_styles( $styles ) {
 		// Until #37466, we can't specifically add them as editor styles yet,
 		// so we must hard-code it here as a dependency.
 		'wp-block-editor-content',
+		'wp-base-styles',
```

Comment:
It feels a bit weird to me that this is added to the blocks editor styles. Logically, it seems better to add as a dependency to all the scripts that do `React.render` instead (top level scripts like edit-post, edit-site, ...)